Branch-SDK/src/io/branch/referral/SystemObserver.java

```diff
@@ -324,7 +324,8 @@ public boolean getBluetoothPresent() {
         try {
             BluetoothAdapter bluetoothAdapter = BluetoothAdapter.getDefaultAdapter();
             if (bluetoothAdapter != null) {
-                return bluetoothAdapter.isEnabled();
+				//noinspection ResourceType
```

Comment:
Oh. There you go. Welcome to your hard tab hell. I actually can't type tabs in my environment.

Because `BluetoothAdapter#isEnabled()` specified 
````
<p>Requires {@link android.Manifest.permission#BLUETOOTH}
````
the warning is
````
Missing permissions required by BluetoothAdapter.isEnabled:  android.permission.BLUETOOTH
````